tools/pybind11Config.cmake.in

```diff
@@ -0,0 +1,91 @@
+# pybind11Config.cmake
```

Comment:
pybind11Config.cmake and pybind11ConfigVersion.cmake should be added to .gitignore so that they don't show up when doing in-tree builds.